spokestack/asr/google/speech_recognizer.py

```diff
@@ -0,0 +1,104 @@
+"""
+This module contains the google asr speech recognizer
+"""
+import logging
+from queue import Queue
+from threading import Thread
+from typing import Any
+
+import numpy as np  # type: ignore
+from google.cloud import speech  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+_LOG = logging.getLogger(__name__)
+
+
+class GoogleSpeechRecognizer:
+    """ Transforms speech into text using Google's ASR.
+
+     Args:
+         cred_path (str): path to google credentials
+         language (str): language of input audio
+         sample_rate (int): sample rate of the input audio (Hz)
+         **kwargs (optional): additional keyword arguments
+     """
+
+    def __init__(
+        self, cred_path, language="en-US", sample_rate=16000, **kwargs,
```

Comment:
we don't want language defaults biased to en-US, this parameter should always be required